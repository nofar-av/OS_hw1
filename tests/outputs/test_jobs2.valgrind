==27523== Memcheck, a memory error detector
==27523== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==27523== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==27523== Command: /home/omer/OS_wet1/smash
==27523== Parent PID: 27519
==27523== 
==27523== 
==27523== FILE DESCRIPTORS: 4 open at exit.
==27523== Open file descriptor 3: /home/omer/OS_wet1/tests/outputs/test_jobs2.valgrind
==27523==    <inherited from parent>
==27523== 
==27523== Open file descriptor 2: /home/omer/OS_wet1/tests/outputs/test_jobs2.err
==27523==    <inherited from parent>
==27523== 
==27523== Open file descriptor 1: /home/omer/OS_wet1/tests/outputs/test_jobs2.out
==27523==    <inherited from parent>
==27523== 
==27523== Open file descriptor 0:
==27523==    <inherited from parent>
==27523== 
==27523== 
==27523== HEAP SUMMARY:
==27523==     in use at exit: 0 bytes in 0 blocks
==27523==   total heap usage: 51 allocs, 51 frees, 83,560 bytes allocated
==27523== 
==27523== All heap blocks were freed -- no leaks are possible
==27523== 
==27523== For counts of detected and suppressed errors, rerun with: -v
==27523== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
