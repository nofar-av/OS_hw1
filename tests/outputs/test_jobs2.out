smash> smash> smash> [1] sleep 10& : 5650 0 secs
smash> smash> [1] sleep 10& : 5650 0 secs
[2] sleep 20& : 5651 0 secs
smash> smash> smash> [1] sleep 10& : 5650 0 secs
[2] sleep 20& : 5651 0 secs
[3] sleep 30& : 5653 0 secs
[4] sleep 40& : 5654 0 secs
smash> smash: sending SIGKILL signal to 4 jobs:
5650: sleep 10&
5651: sleep 20&
5653: sleep 30&
5654: sleep 40&
